src/hooks/transactions/useTransactionType.ts

```diff
@@ -53,11 +55,14 @@ export default function useTransactionType({ input, type, weightInput }: Props):
     weight: null
   });
 
+  const planck = 10 ** targetApi.registry.chainDecimals[0];
+  const [resInp, msg] = evalUnits((parseFloat(input) * planck).toString());
```

Comment:
This `parseFloat` and multiplication should be done in big number context already (same inside `evalUnits`). While it most likely doesn't crash it will loose precision here. Try something like `1.00...001` with large amount of zeros:)